src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx

```diff
@@ -21,6 +21,8 @@ import {
   PopularSpecies
 } from 'src/content/app/species-selector/types/species-search';
 
+import GoogleAnalyticsTracking from 'src/services/analytics-service';
```

Comment:
Can you call it analyticsService (throughout the PR)? The service acts as an interface between the code that sends analytics events and the actual analytics backend to which we are sending this event. If we change from GoogleAnalytics to any other service, we won't need to update files that use the `analytics-service` file.

Reply:
That's done.